subsys/bluetooth/controller/ll_sw/ctrl.c

```diff
@@ -1982,11 +1982,17 @@ isr_rx_conn_pkt_ctrl_rej_conn_upd(struct radio_pdu_node_rx *node_rx,
 		}
 	}
 
+	/* Procedure complete */
```

Comment:
As it was a LL same procedure collision, the local device would keep the CPR context and timeout. The peer shall complete the connection parameter request procedure hence it is not required to reset the state and timeout.

Could you give details of the issue leading to this PR?

Reply:
Analyzed the sniffer logs. This PR is to workaround peer implementation that dont consider at Connection Update Procedure (CUP) instant the procedure being complete.

Hence move the code from line 1971-1977 (reset ack = req and procedure_expire = 0) to line 1985 here.

Reply:
done